mkdocs/docs/HPC/alphafold.md

```diff
@@ -0,0 +1,232 @@
+So that you don't have to download that data yourself.
+
+Under: `{{directory}}`
+
+```shell
+$ ls /arcanine/scratch/gent/apps/AlphaFold
+20210812  20211201  20220701  20230310
+```
+
+The directories located there indicate when the data was downloaded, so that this leaves room for providing updated datasets later.
+
+As of writing this documentation the latest version is `20230310`.
+
+One important remark here is that the filesystem where this AlphaFold dataset is located is *not* available on the default victini cluster,
+as this cluster doesn't have a fast connection to the shared storage.
+
+!!! Info
+    
+    The `arcanine scratch` shared filesystem is powered by fast SSD disks, 
+    which is recommended for the AlphaFold data, because of random access I/O patterns.
+    See [Pre-defined user directories](http://localhost:8000/HPC/Gent/running_jobs_with_input_output_data/#pre-defined-user-directories) to get more info about the arcanine filesystem.
+
+The AlphaFold installations we provide have been modified a bit to facilitate the usage on {{hpcinfra}}.
+
+### Setting up the environment
+
+The location to the AlphaFold data can be specified via the `$ALPHAFOLD_DATA_DIR` environment variable, so you should define this variable in your AlphaFold job script:
+
+```shell
+export ALPHAFOLD_DATA_DIR={{directory}}/{{version}}
+```
+
+!!! Warning "Use newest version"
+    
+    Do not forget to replace `{{version}}` with a more up to date version if available.
+
+### Running AlphaFold
+
+AlphaFold provides run script called [run_alphafold.py](https://raw.githubusercontent.com/deepmind/alphafold/main/run_alphafold.py)
+
+A symbolic link named *alphafold* that points to the this script is included,
+so you can just use `alphafold` instead of `run_alphafold.py` or `python run_alphafold.py` after loading the AlphaFold module.
+
+`run_alphafold.py` script has also been slightly modified such that defining the `$ALPHAFOLD_DATA_DIR` (see [above](./#setting-up-the-environment)) is sufficient to pick up all the data provided in that location, 
+so you don't need to use options like `--data_dir` to specify the location of the data.
+
+Similarly, the script was also tweaked such that the location to commands like `hhblits,hhsearch,jackhmmer,kalign` are already correctly set, so options like `--hhblits_binary_path` are **not** required.
+
+For more information about the script and options see [this section]({{readme}}#running-alphafold) in the official [README]({{readme}}).
+
+!!! WARNING "READ README"
+
+    It is **strongly** advised to read the official [README]({{readme}}) provided by deepmind before continuing.
+    
+
+### Tweaking hhblits and jackhmmer cpu cores
+
+The Python script that are used to run ***hhblits and jackhmmer*** have been tweaked so you can control how many cores are used for these tools,
+rather than hardcoding it to 4 and 8 cores, respectively.
```

Comment:
```suggestion
The Python scripts that are used to run ***hhblits and jackhmmer*** have been tweaked so you can control how many cores are used for these tools,
rather than hardcoding it to 4 and 8 cores, respectively.